### PR TITLE
Add Psort task

### LINKED ISSUE #18: Add psort output to plaso plugin
Output in CSV format.  Also Create new evidence type for PlasoCSVFile.

### PR DESCRIPTION
**Pull request by aarontp**
Fixes #18

### CODE REVIEW

File: turbinia/workers/psort.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+from turbinia.workers import TurbiniaTask
+from turbinia.evidence import PlasoCsvFile
+
+
+class PsortTask(TurbiniaTask):
+  """Task to run Psort to generate CSV output from plaso files."""
+
+  def run(self, evidence, result):
+    """Task that processes Plaso files with Psort.
```
Comment (berggren) [inline turbinia/workers/psort.py]: Nit: Plaso storage files

File: turbinia/evidence.py [hunk 2]
```diff
@@ -257,6 +257,15 @@ def __init__(self, plaso_version=None, *args, **kwargs):
     super(PlasoFile, self).__init__(*args, **kwargs)
 
 
+class PlasoCsvFile(PlasoFile):
+  """Psort output file evidence.  """
+
+  def __init__(self, plaso_version=None, *args, **kwargs):
+    """Initialization for Plaso File evidence."""
+    self.plaso_version = plaso_version
```
Comment (berggren) [inline turbinia/evidence.py]: Where do you get plaso_version from?

Comment (aarontp) [reply turbinia/evidence.py]: Right now I don't, but there's a TODO in the plaso task code, :)

File: turbinia/workers/psort.py (hunk 1 above)
Comment (aarontp) [reply turbinia/workers/psort.py]: Fixed.

**Comment by aarontp** [comment]
PTAL

File: turbinia/evidence.py (hunk 2 above)
Comment (berggren) [reply turbinia/evidence.py]: ack

**Pull request merged**